packages/web-wallet/src/dataProviders/identifiersDataProvider.ts

```diff
@@ -137,7 +137,7 @@ export const identifiersDataProvider = (): DataProvider => ({
         clientId,
         credentialIssuer: ebsi?.tao?.url!,
         jwksUri,
-        environment: network as EbsiEnvironment,
+        environment: network as EbsiEnvironment, //FIXME this is casting a possible undefined where environment is mandatory
```

Comment:
Shouldn't you throw an error then when (under method === 'ebsi') network is undefined? Or should we really be able to pass as undefined?